src/main.rs

```diff
@@ -101,7 +101,11 @@ fn load_config(options: &cli::Options) -> Config {
         });
 
     Config::load_from(&*config_path).unwrap_or_else(|err| {
-        error!("Error: {}; Loading default config", err);
+        match err {
+            ConfigError::Empty => info!("Config file: {}; Loading default config", err),
```

Comment:
Now I'm going to border on irreverent bike shedding, but I'd prefer to avoid duplicate words, and extraneous symbols.

```
[2018-11-23 14:11] [INFO] Config file: empty config file; Loading default config
vs
[2018-11-23 14:11] [INFO] Config file is empty, loading default
```

Reply:
Hmmmm, yea this is generally an interesting issue of how to reuse error strings in different grammatical situations. I'm personally of the opinion the result (error message in this case) shouldn't be driven by implementation details, but I concede some value in using the existing error message.

Reply:
I'm perfectly fine with throwing away the error message if it's easier to write it completely from scratch. It's supposed to help and make things easier, not the other way around. If there's no useful info in there, get rid of it. Since we already know exactly what the error is anyways by matching it, there's not much need to re-use it. Alternatively the error message itself can also be altered.

I'd probably go for something like this as a target output:
```
[2018-11-23 15:99] [INFO] configuration file is empty; loading default
```

Capitalization is mainly because that's what all other errors look like too iirc.

Reply:
Actually most messages are capitalized, but this isn't very consistent.

This level of nit-picking would probably be best addressed all at once across the whole repo, though who cares?

Reply:
Running `grep -Rsi "error\!" src` in this repo seems to prove otherwise. Almost all of these logs start lowercase. The letter after the `;` is capitalized though, I didn't really look at that.

Reply:
That's why I added the (like this), since in fact most log messages are printing objects attributes in lowercase, custom messages seem to be capitalized. For example:

https://github.com/jwilm/alacritty/blob/master/src/main.rs#L120

Reply:
Agree 100%, I'll have a PR up soon normalizing the messaging. I can easily rebase master once this is merged and any further discussion can happen on the PR.